### ISSUE TITLE
Incorrect scaling for CFG_SFIMU_IMU_MNTALG* config items

### ISSUE THREAD

**Issue by bvsam** [aussierobots/ublox_dgnss#31]
The `scale` member of the `CFG_SFIMU_IMU_MNTALG_YAW`, `CFG_SFIMU_IMU_MNTALG_PITCH`, and `CFG_SFIMU_IMU_MNTALG_ROLL` structs (all of type `ubx_cfg_item_t`) in `ublox_dgnss_node/include/ublox_dgnss_node/ubx/ubx_cfg_item.hpp` is incorrect. They're all set to `.001` (which is `1e-3`), when they should be set to `1e2`, according to the [`F9 HPS 1.30 Interface description manual`](https://www.u-blox.com/sites/default/files/documents/u-blox-F9-HPS-1.30_InterfaceDescription_UBX-22010984.pdf) (which seems to be the latest version).

This seems to be a very minor issue, since the `scale` member is not used anywhere else in the code, but just mentioning it for reference.

**Comment by hortovanyi** [comment]
Yes you are correct. Must have been a cut and paste error. Its used with the F9R. 

The scale in this case is what you should multiply it by for writing out to log files
